pkgs/applications/misc/oil/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, jq, peco, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "oil-buku";
+  version = "0.3.2";
+
+  src = fetchurl {
+    url = "https://github.com/AndreiUlmeyda/oil/archive/refs/tags/${version}.tar.gz";
+    sha256 = "1nwas1v0z2sjfab8ada9k9pf4jw4jn79rzmmn4n69xlfzzf4npq6";
+  };
+
+  passthru.updateScript = [ ./update.sh ];
+
+  postUnpack = ''
+    sed -i 's/$LIBDIR/$LOCALLIB/g' oil-${version}/src/oil
+  '';
+
+  buildInputs = [ jq peco ];
```

Comment:
Since these get wrapped directly into `$PATH` already, you don't need to set `buildInputs`. That is only used at build time, see https://nixos.org/manual/nixpkgs/unstable/#ssec-stdenv-dependencies

Reply:
Thx for the input.